src/third_party/wiredtiger/src/include/misc.h

```diff
@@ -85,7 +85,7 @@
  * Default hash table size; we don't need a prime number of buckets
  * because we always use a good hash function.
  */
-#define	WT_HASH_ARRAY_SIZE	512
+#define	WT_HASH_ARRAY_SIZE	65536
```

Comment:
I think we should go to 128k because usually we see active dhandles go up to 100k.
I remember also that it is usually ideal to use 2x the number of entries if possible to avoid most collisions.
Looking online, some people recommend 1.2-1.3-1.5x at least but not sure there is a good explanation.

Reply:
Ok